Sprint-2/implement/to-pounds.js

```diff
@@ -4,3 +4,13 @@
 // You will need to declare a function called toPounds with an appropriately named parameter.
 
 // You should call this function a number of times to check it works for different inputs
+
+
+function toPounds(kilograms) {
```

Comment:
#### Todo
If you refer to `interpret/to-pounds.js` in Sprint 1, you will find the code there is to convert pence string in the form "399p" to British pound "£3.99". This function should be based on those code.